src/shared/logs-show.c

```diff
@@ -317,15 +317,47 @@ static bool print_multiline(
         return ellipsized;
 }
 
-static int output_timestamp_monotonic(FILE *f, const dual_timestamp* const ts) {
+static int output_timestamp_monotonic(
+                FILE *f, OutputMode mode,
+                const dual_timestamp* const ts,
```

Comment:
please drop the second `const`.

There's no value in making function parameters const, they are copied anyway...

Making the target of a pointer parameter const makes a ton of sense, since it allows indicating what is changed and what isn't, for *callers* of the function. But if you add the const to the pointer itself instead of the pointer target this is just noise, since it has zero effect for callers, and is purely an internal thing.

hence, please keep the `const` on the target (i.e. the first `const`), drop the one on the pointer (i.e. the second `const`).

same for the other three arguments

Reply:
Done